### PR TITLE
[skin.confluence] Add 'Get more' to media menus

### PR DESCRIPTION
**Pull request by Hitcher**
For when they're removed from the VFS.

@tamland RE https://github.com/xbmc/xbmc/pull/8716

### CODE REVIEW

**Comment by HitcherUK** [comment]
screenshot001

File: addons/skin.confluence/720p/MyMusicNav.xml [hunk 1]
```diff
@@ -148,6 +148,14 @@
 					<alttexturenofocus>-</alttexturenofocus>
 					<usealttexture>library.isscanningmusic</usealttexture>
 				</control>
+				<control type="button" id="204">
+					<description>Get more</description>
+					<textwidth>235</textwidth>
+					<include>ButtonCommonValues</include>
+					<label>21452</label>
+					<onclick>ActivateWindow(addonbrowser,addons://all/xbmc.addon.audio/,return)</onclick>
+					<visible>StringCompare(Container.FolderName,$LOCALIZE[24001]</visible>
```
Comment (phil65) [inline addons/skin.confluence/720p/MyMusicNav.xml]: missing closing bracket

File: addons/skin.confluence/720p/MyPics.xml [hunk 2]
```diff
@@ -132,6 +132,23 @@
...
+					<align>center</align>
+					<aligny>center</aligny>
+				</control>
+				<control type="button" id="204">
+					<description>Get more</description>
+					<textwidth>235</textwidth>
+					<include>ButtonCommonValues</include>
+					<label>21452</label>
+					<onclick>ActivateWindow(addonbrowser,addons://all/xbmc.addon.image/,return)</onclick>
+					<visible>StringCompare(Container.FolderName,$LOCALIZE[24001]</visible>
```
Comment (phil65) [inline addons/skin.confluence/720p/MyPics.xml]: same here

File: addons/skin.confluence/720p/MyPrograms.xml [hunk 3]
```diff
@@ -90,6 +90,23 @@
...
+					<align>center</align>
+					<aligny>center</aligny>
+				</control>
+				<control type="button" id="204">
+					<description>Get more</description>
+					<textwidth>235</textwidth>
+					<include>ButtonCommonValues</include>
+					<label>21452</label>
+					<onclick>ActivateWindow(addonbrowser,addons://all/xbmc.addon.executable/,return)</onclick>
+					<visible>StringCompare(Container.FolderName,$LOCALIZE[24001]</visible>
```
Comment (phil65) [inline addons/skin.confluence/720p/MyPrograms.xml]: here

File: addons/skin.confluence/720p/MyVideoNav.xml [hunk 4]
```diff
@@ -170,6 +170,14 @@
 					<alttexturenofocus>-</alttexturenofocus>
 					<usealttexture>library.isscanningvideo</usealttexture>
 				</control>
+				<control type="button" id="204">
+					<description>Get more</description>
+					<textwidth>235</textwidth>
+					<include>ButtonCommonValues</include>
+					<label>21452</label>
+					<onclick>ActivateWindow(addonbrowser,addons://all/xbmc.addon.video/,return)</onclick>
+					<visible>StringCompare(Container.FolderName,$LOCALIZE[24001]</visible>
```
Comment (phil65) [inline addons/skin.confluence/720p/MyVideoNav.xml]: here :)

**Comment by phil65** [comment]
I agree with ronie, container.content(addons) instead of stringcompare() would be nicer for the media windows.

**Comment by phil65** [comment]
...and perhaps also exchange the label "get more..." to sth different? the context changed a bit.

**Comment by HitcherUK** [comment]
What label do you suggest? I want to use an existing one if possible to save time.

**Comment by phil65** [comment]
hmm no idea atm, sorry.
 just keep it as it is then, that´s why I used the word "perhaps" :)

File: addons/skin.confluence/720p/MyMusicNav.xml [hunk 5]
```diff
@@ -148,6 +148,14 @@
 					<alttexturenofocus>-</alttexturenofocus>
 					<usealttexture>library.isscanningmusic</usealttexture>
 				</control>
+				<control type="button" id="204">
+					<description>Get more</description>
+					<textwidth>235</textwidth>
+					<include>ButtonCommonValues</include>
+					<label>21452</label>
+					<onclick>ActivateWindow(addonbrowser,addons://all/xbmc.addon.audio/,return)</onclick>
```
Comment (tamland) [inline addons/skin.confluence/720p/MyMusicNav.xml]: Please use `addons://more/audio` (etc) paths similar to addons://sources/*. The 'all' paths are intended for internal use

**Comment by tamland** [comment]
Thanks @HitcherUK !

File: addons/skin.confluence/720p/MyMusicNav.xml (hunk 5 above)
Comment (HitcherUK) [inline addons/skin.confluence/720p/MyMusicNav.xml]: @tamland Neither `addons://more/audio/` or `addons://more/xbmc.addon.audio/` work.

Comment (tamland) [inline addons/skin.confluence/720p/MyMusicNav.xml]: Yeah, typo by me. I wrote 'music' for some reason. The other ones works though, correct? (you'll need #8716)

Comment (HitcherUK) [inline addons/skin.confluence/720p/MyMusicNav.xml]: Ah, that'll be why they don't work then :)
Just fired off a test build.

**Comment by HitcherUK** [comment]
@tamland Tested with `addons://more/` and all work OK but I can't test Music Addons because there's no node for them.

**Comment by tamland** [comment]
"I can't test Music Addons because there's no node for them." <- not sure what that means. Tested current code with #8716 and it all seems to work fine here..

**Comment by tamland** [comment]
@HitcherUK Good to go? (I guess you should squash?)

**Comment by HitcherUK** [comment]
@tamland See https://github.com/xbmc/xbmc/pull/8827

**Pull request closed without merging**